ios/RNShare.m

```diff
@@ -104,12 +104,12 @@ - (NSDictionary *)constantsToExport
 {
   return @{
     @"FACEBOOK": @"facebook",
-    @"FACEBOOK_STORIES": @"facebookstories",
+    @"FACEBOOKSTORIES": @"facebookstories",
```

Comment:
Why did this change?

Reply:
To standardize naming between iOS and Android. 
In android we've got `facebookstories` (which is later uppercased to `FACEBOOKSTORIES`).
Original implementation -> `NativeModules.RNShare.FACEBOOK_STORIES` was returning undefined on Android (because the key name there is `FACEBOOKSTORIES`).